webpack.config.js

```diff
@@ -17,6 +17,14 @@ module.exports = {
             presets: ['@babel/preset-env', '@babel/preset-react']
           }
         }
+      },
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+          },
```

Comment:
What files are you intending to load? I just stuck everything in my S3 bucket and use URLs so I don't have to serve anything up from my server aside from the html and js files, though this isn't the only way to do it.

Reply:
@CSellers720 - This is a good example of when I almost don't want a code review; I was just testing out a feature in my repo. If I was more savvy with git, I would have created that sub-branch, experimented, and then discarded/merged accordingly. Alas, I can only consume so much from the fire hose at once ;)